lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,79 @@
+defmodule SituationRoom.Site.Controller do
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+  # If we are going to include plug.Parsers, they go right here
```

Comment:
These kind of comments can be removed, as they appear to be more debug/informational to you personally but not applicable to the code project documentation as a whole.